src/test/kotlin/com/zepben/evolve/services/network/translator/NetworkTranslatorTest.kt

```diff
@@ -149,6 +150,7 @@ internal class NetworkTranslatorTest {
         /************ IEC61970 InfIEC61970 FEEDER ************/
         validate({ Circuit() }, { ns, it -> it.fillFields(ns) }, { ns, it -> ns.addFromPb(it.toPb()) })
         validate({ Loop() }, { ns, it -> it.fillFields(ns) }, { ns, it -> ns.addFromPb(it.toPb()) })
+        validate({ LvFeeder() }, { ns, it -> it.fillFields(ns)}, { ns, it -> ns.addFromPb(it.toPb()) })
```

Comment:
You have this test which should be checking from CIM to protobuf back to CIM but several parts were not covered so something seems to not be working properly here.